source/commands.txt

```diff
@@ -0,0 +1,163 @@
+       directly into the command palette.
+
+       |service| provides a connection string for your clusters which
+       you can pass into this command.
+       
+       To learn more about connecting to a deployment, see
+       :ref:`vsce-connect`.
+
+   * - :ref:`Disconnect <vsce-cmd-disconnect>`
+
+       .. _vsce-cmd-disconnect:
+
+     - If you are currently connected to a MongoDB deployment,
+       disconnect from that deployment.
+
+   * - :ref:`Remove Connection <vsce-cmd-remove-connection>`
+
+       .. _vsce-cmd-remove-connection:
+
+     - Select and remove a connection from your list of connections
+       in |vsce|.
+
+.. note::
+
+   You can only have one active connection to a MongoDB
+   deployment open at a time.
+
+Shell Commands
+--------------
+
+Run this command from the Command Palette to open a MongoDB Shell
+connected to your deployment.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 40
+
+   * - Command
+     - Description
+
+   * - :ref:`Launch MongoDB Shell <vsce-cmd-launch-shell>`
+
+       .. _vsce-cmd-launch-shell:
+
+     - .. important::
+
+          This command is not supported on Windows.
+     
+       Open the VS Code :guilabel:`Terminal` and automatically connect a
+       MongoDB shell to the deployment where |vsce| is connected. You
+       can set which shell this command uses in your ``mdb.shell``
+       :ref:`setting <vsce-settings>`.
+
+       To run this command, you must have either the legacy
+       :binary:`~mongo` shell or ``mongosh`` shell installed.
+
+       To learn how to install the legacy :binary:`~mongo` shell, see
+       :manual:`Instal MongoDB </installation>`.
+
+Playground Commands
+-------------------
+
+Run these commands from the Command Palette to create and run
+:ref:`playgrounds <vsce-playgrounds>`.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 40
+
+   * - Command
+     - Description
+
+   * - :ref:`Create MongoDB Playground <vsce-cmd-playground-create>`
+
+       .. _vsce-command-playground-create:
+
+     - Create a new playground to run against your MongoDB deployment.
+
+       By default, playgrounds load with a template containing
+       sample commands. You can disable this template from your
+       :ref:`extension settings <vsce-settings>`.
+
+   * - :ref:`Run All From Playground <vsce-cmd-playground-run>`
+
+       .. _vsce-command-playground-run:
+
+     - Run file you are currently viewing in VS Code as a playground.
+       |vsce| outputs the results of your playground in the
+       :guilabel:`Output` view in VS Code.
+
+|vsce| View Commands
+--------------------
+
+Run these commands from the Command Palette to manage the |vsce| view in
+the left navigation panel.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 40
+
+   * - Command
+     - Description
+
+   * - :ref:`Focus on MongoDB View <vsce-cmd-focus>`
+
+       .. _vsce-cmd-focus:
+
+     - Open the |vsce| view and change the focus of VS Code to
+       the extension.
```

Comment:
Maybe "Open and focus on the |vsce| view?

Reply:
I wasn't sure if "focus" would be commonly understood when phrased as such, but your suggestion is definitely cleaner so I'll go with it.